spec/formalism/form/coercion_spec.rb

```diff
@@ -0,0 +1,199 @@
+# frozen_string_literal: true
+
+## This is unit test, created to simplify testing of different coercions
+## You can find integration test in spec/formalism/form/fields_spec.rb
+## (Formalism::Form -> .field -> coersion)
+##
+## In tests `eql` used instead of `eq` to determine result type
+describe 'Formalism::Coercion' do
+	let(:value) {}
+
+	let(:field_defenition) { [type] }
+	let(:form) do
+		## Pass method call result throug instance_eval inside Class.new
+		field_defenition = self.field_defenition
+
+		Class.new(Formalism::Form) do
+			field :value, *field_defenition
```

Comment:
I think, `field_type` is better.

Reply:
Don't sure `field_type` is an array, but as you wish